src/components/KCard/KCard.vue

```diff
@@ -42,6 +43,10 @@ const props = defineProps({
     type: String,
     default: '',
   },
+  titleTag: {
+    type: String,
```

Comment:
question: should you do any validation on the prop value? I'd think you could have a validator that only allows these:
```md
['div', 'span', 'p', 'a', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6',]
```

Reply:
Doen